apis/externalsecrets/v1alpha1/pushsecret_types.go

```diff
@@ -106,6 +106,7 @@ type PushSecretData struct {
 }
 
 // PushSecretConditionType indicates the condition of the PushSecret.
+// +kubebuilder:validation:Enum=Ready
```

Comment:
The same as https://github.com/external-secrets/external-secrets/pull/2860/files#r1385652402